docs/components/optimize/userguide/additional-features/filters.md

```diff
@@ -0,0 +1,235 @@
+
+This filter type can be applied either as a [Process Instance](#filter-behavior) or as a [flow node](#filter-behavior) filter: 
+
+- When applied as a process instance filter, you are required to select the flow nodes that are to be relevant to the filter, yielding a report which will only consider those process instances where one or more of the selected flow nodes match the configured filter.
+
+![Flow Node date filter](./img/flowNode-date-filter.png)
+
+- When added as a flow node filter, there is no flow node selection. The resulting report automatically only includes data from those flow nodes which match the given filter.
+
+## Duration filters 
+
+### Process instance duration filter
+
+The `Process Instance Duration Filter` allows you to only regard process instances whose execution from start to end took a certain amount of time. For instance, you can filter process instances that took more than three days or less than five seconds.
+
+:::note
+This filter shows only completed process instances, since the total duration of running process instances is not yet known.
+:::
+
+![Process instance duration filter in Camunda Optimize](./img/duration-filter.png)
+
+### Flow node duration filter
+
+If the `Flow Node Duration Filter` is applied as an instance filter, it will only regard process instances where one or more flow nodes took a certain amount of time for their execution. For instance, you can filter process instances where a flow node took more than three days or less than five seconds. 
+
+If applied as a flow node filter, it will filter flow nodes and only show the flow nodes that were selected in the filter.
+
+![Flow Node duration filter in Camunda Optimize](./img/flowNode-duration-filter.png)
+
+:::note
+For incident reports, flow node duration filters always behave as instance filters regardless of where they were defined. 
+:::
+
+## Flow node filter
+
+Retrieve only those process instances that executed certain flow nodes within your process by using the `Flow Node Filter`. Selecting several values at once means that all the selected flow nodes need to have been executed at least once in the process instance lifetime. At the top of the flow node filter modal you can see a preview of the filter you are about to create. You can also filter process instances where certain flow nodes were not executed.
+
+![Flow node filter in Camunda Optimize](./img/flownode-filter.png)
+
+## Flow node selection
+
+In flow node and user tasks reports, all flow nodes are included in the result by default. This could result in a lot of table rows or chart entries which makes the visualization hard to read. This filter allows you to specify which flow nodes are included and deselect the ones that are not relevant to the report.
+
+![Specifying which nodes are included in the report](./img/flowNodeSelection.png)
+
+## Variable filter
```

Comment:
This section looks super intimidating. If you are running locally see - http://localhost:3000/docs/components/optimize/userguide/additional-features/filters/#variable-filter

Can you break down the 4 types of variables into header sections instead of an unordered list? Looks like they would be `###H3`'s

Reply:
Agreed. Quite frankly, this doc in its entirety is very intimidating. Could be worthwhile to give this a good look in Q1 and consider a restructure?